### PR TITLE
beep: 1.3 -> 1.4.9

### PR DESCRIPTION
**Pull request by freezeboy**
###### Things done

- [X] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [X] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/misc/beep/default.nix [hunk 1]
```diff
@@ -1,25 +1,24 @@
...
 
-stdenv.mkDerivation {
-  name = "beep-1.3";
-  src = fetchurl {
-    url = "http://www.johnath.com/beep/beep-1.3.tar.gz";
-    sha256 = "0bgch6jq5cahakk3kbr9549iysf2dik09afixxy5brbxk1xfzb2r";
+stdenv.mkDerivation rec {
+  pname = "beep";
+  version = "1.4.9";
+  src = fetchFromGitHub {
```
Comment (SuperSandro2000) [inline pkgs/misc/beep/default.nix:9]:
```suggestion
  version = "1.4.9";

  src = fetchFromGitHub {
```

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 107729` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 107729` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>beep</li>
  </ul>
</details>

`gpl2` is a deprecated license, check if project uses `gpl2Plus` or `gpl2Only` and change `meta.license` accordingly.

**Comment by freezeboy** [comment]
Both done, I thought I had fixed the license before, sorry

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 107729` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>beep</li>
  </ul>
</details>

**Pull request merged**